tests/static/vasp_test_files/poscar_samples/POSCAR_velocity

```diff
@@ -0,0 +1,692 @@
+Poscar file generated with pyiron       
```

Comment:
I guess the file should not have more than 50 lines

Reply:
https://github.com/pyiron/pyiron/pull/133/commits/ee1079a0cea83405664e09827befd4c0211d6489